CHANGELOG.md

```diff
@@ -98,6 +98,8 @@
   ([#6221](https://github.com/facebook/jest/pull/6221))
 * `[expect]` Improve return matchers
   ([#6172](https://github.com/facebook/jest/pull/6172))
+* `[jest-mock]` Include tracked call results in serialized mock
```

Comment:
should probably be marked as a breaking change in hindsight - this will break people's snapshots if the have jest mocks in them

Reply:
Just experienced it. This is definitely a breaking change.